src/mixins/test/triangle.test.js

```diff
@@ -267,7 +267,7 @@ describe('triangle', () => {
     )
   })
 
-  it('should throw an error when height or width is not a unit based value.', () => {
+  it.skip('should throw an error when height or width is not a unit based value.', () => {
```

Comment:
Q for the reviewer, do we want to keep this check, but then also explicitly check for a value starting with `var(--`?